drafts/use-cases/1.entry-point.md

```diff
@@ -28,18 +28,27 @@ HTTP 200 OK
 ```
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@id": "/api",
     "@type": "hydra:EntryPoint",
-    "events": {
-        "@id": "/api/events", 
+    "#events": {
+        "@id": "/api/events",
         "title": "List of events",
         "@type": "hydra:Collection",
-        "operation": {
-            "@type": "hydra:Operation",
-            "title": "Create new event",
-            "method": "POST",
-            "expects": "schema:Event"
-        }
+        "operation": [
+            {
+               "@type": ["hydra:Operation", "schema:ViewAction"],
+                "title": "List events",
+                "method": "GET",
```

Comment:
> Hydra is based on the open world assumption

Well sure, no one stops your from dereferencing any resource. But as the client (library), I'd expect clear hints on what the intended usage is. Otherwise you could defend a stance that any HTTP request is valid and the client would brute-force anything. Especially that Hydra has no way to mark operations as forbidden